lib/verticalresponse/api/user.rb

```diff
@@ -0,0 +1,32 @@
+# Class that represents a user account info resource from the VerticalResponse API.
+# It has the ability to make REST calls to the API, as well as to wrap
+# the contact objects we get as response.
+#
+# NOTE: This class does not necessarily include all the available methods
+# the API has for the contact resource. You can consider this an initial
+# approach for an object oriented solution that you can expand according to
+# your needs.
+require_relative 'client'
+
+module VerticalResponse
+  module API
+    class User < Resource
+
+      class << self
+        def resource_uri_suffix
+          ['users']
+        end
+
+        def initialize(*args)
+          super
+        end
+
+        def info(options = {})
+          url = resource_uri + "/" + options['uid'].to_s
```

Comment:
Interpolate this string instead. Also, you won't need the `#to_s` anymore since it's called automatically during interpolation.